fix for a segfault on:
format->character->hyperlink->events->oo.org macros->launcher

Index: source/dialog/cfg.cxx
===================================================================
RCS file: /cvs/framework/sfx2/source/dialog/cfg.cxx,v
retrieving revision 1.53
diff -u -p -u -r1.53 cfg.cxx
--- sfx2/source/dialog/cfg.cxx	7 Sep 2005 18:12:28 -0000	1.53
+++ sfx2/source/dialog/cfg.cxx	14 Mar 2006 10:50:22 -0000
@@ -1746,7 +1746,7 @@ void SfxConfigGroupListBox_Impl::Request
 									bDisplay=FALSE;
 								}
 							}
-							if (children[n]->getType() != browse::BrowseNodeTypes::SCRIPT  && bDisplay )
+							if ( children[n].is() && children[n]->getType() != browse::BrowseNodeTypes::SCRIPT && bDisplay )
 							{
 
 								/*
